build.sh

```diff
@@ -184,9 +186,14 @@ if (( ${NUMARGS} == 0 )) || hasArg libcuml || hasArg prims || hasArg bench; then
 
 fi
 
+if hasArg cppdocs; then
+    cd ${LIBCUML_BUILD_DIR}
```

Comment:
nitpick - use `pushd <somedir>` then run `popd` after make doc so you get returned to your previous directory.